components/Board.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Letter from "./Letter";
+
+export default function Board() {
+  return (
+    <div className="board">
+      {" "}
+      <div className="row">
+        <Letter rowIndex={0} letterIndex={0} />
```

Comment:
The components in this section look a little bit repetitive -- it looks like it would be possible to build out these rows and programmatically set the indexes using some sort of a loop (map or foreach).